find_within_files.sh

```diff
@@ -63,6 +76,12 @@ if [[ "$PREVIEW_ENABLED" -eq 1 ]]; then
     PREVIEW_STR=(--preview "$PREVIEW_COMMAND" --preview-window "$PREVIEW_WINDOW")
 fi
 
+RESUME_POS_BINDING="start:ignore" # dummy fallback binding because fzf v<0.36 does not support `load` and I did not figure out how to conditionally set the entire binding string (i.e., with the "--bind" part)
```

Comment:
Yeah, this is the way. Bash is a terrible, terrible language for stuff like this, and I really feel like I've been stretching what's within reason to do with bash while working on this extension... It's been painful :')

Reply:
I tried this and I actually get `unsupported key: start` now :) I had luck changing it to `backward-eof:ignore`. Definitely a hack but it seems to work :) I'm going to merge this and add this in.